catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -142,6 +142,10 @@ public CreateStorageResponse create(CreateStorageRequest createRequest)
     public ReadStorageResponse read(ReadStorageRequest readRequest) throws StorageException {
         LOGGER.trace("ENTERING: read");
 
+        if (readRequest.getResourceUri() == null) {
+            return new ReadStorageResponseImpl(readRequest);
```

Comment:
Checked DDF and a little bit of downstream code and didn't find anybody really using it.